readme.md

```diff
@@ -17,6 +17,12 @@ install packages:
 brew install node docker postgres dbmate
 ```
 
+(optional) install prisma studio
```

Comment:
do you have to install something here? Why can't you just run `yarn orm studio` or whatever

Reply:
Oh - you can do that, just checked. Didn't realize you could open the studio from the command line - I'll update the docs with this change and the change for your comment above